Python/Product/PythonTools/PythonTools/Intellisense/IntellisenseController.cs

```diff
@@ -241,7 +241,9 @@ internal sealed class IntellisenseController : IIntellisenseController, IOleComm
                 }
 
                 if (entry == null) {
-                    Debug.Fail($"Failed to analyze {buffer.DocumentUri}");
+                    if (!_services.Python.LanguageServerOptions.ServerDisabled) {
```

Comment:
Should we just avoid attaching the IntellisenseController completely when disabled?